hazelcast/src/main/java/com/hazelcast/partition/impl/ReplicaSyncInfo.java

```diff
@@ -18,7 +18,7 @@
 
 import com.hazelcast.nio.Address;
 
-final class ReplicaSyncInfo {
+public final class ReplicaSyncInfo {
```

Comment:
It seems it's imported for only test classes, instead of making this public, you can move tests into impl package.